types/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// TypeScript Version: 3.5
+
+import {Node} from 'unist'
+import {Test} from 'unist-util-is'
+
+// NOTE: namespace is needed to use `export = remove`
+declare namespace remove {}
+
+declare interface RemoveOptions {
+  /**
+   * Whether to drop parent nodes if they had children, but all their children were filtered out test
+   */
+  cascade?: boolean
+}
```

Comment:
This should be moved into the namespace, so this type can be used as well.

```suggestion
declare namespace remove {
  interface RemoveOptions {
    /**
     * Whether to drop parent nodes if they had children, but all their children were filtered out test
     */
    cascade?: boolean
  }
}
```

Apart from that, looks good to me :+1: